### ISSUE TITLE
Case Sensitive tvg-id

### ISSUE THREAD

**Issue by ap-rose** [jjssoftware/m3u-epg-editor#32]
Could you please add an argument to be able to match tvg-id regardless of case.

I am having issues where my EPG contains data that is not being kept due to id being case sensitive.

All EPG providers change the format of their ID making it hard to parse properly.

**Comment by jjssoftware** [comment]
I haven't experienced this issue myself but I'll take a look to see if I can reproduce the problem and come up with something to cope with it.

**Comment by ap-rose** [comment]
The issue I am having is that if the m3u contain the tvg-id as lowercase (tlc.ca) it will not be matched to my data found in my epg which is defined in uppercase. (TLC.ca)

Is there a way too loosen the match so it is not case sensitive.

**Comment by jjssoftware** [comment]
Yep I get the idea. I've created a branch and applied a candidate change:

https://github.com/jjssoftware/m3u-epg-editor/tree/epg-tvg-id-casing

Give this version a try and let me know how it goes:

https://github.com/jjssoftware/m3u-epg-editor/blob/epg-tvg-id-casing/m3u-epg-editor.py

**Comment by ap-rose** [comment]
The fix you submitted did the trick, Thank you so much for prompt response.

Just know your software was of great use to me and continuing to use it.

**Comment by jjssoftware** [comment]
You're welcome, thanks for the update, I'll merge the change.

**Comment by jjssoftware** [comment]
Merged to master in https://github.com/jjssoftware/m3u-epg-editor/pull/33
